src/internal/KeyPair.ts

```diff
@@ -67,3 +68,9 @@ export class KeyPair {
         return this.Libp2pPeerId.privKey.public.verify(data, signature);
     }
 }
+
+export const keyPairFromBase64Sk = async (sk: string) => {
+    const arr = toUint8Array(sk);
+    const kp = await KeyPair.fromEd25519SK(arr);
+    return kp;
```

Comment:
Even though it's negligable - creation of additional promise can be avoided if we replace this with just

return KeyPair.fromEd25519SK(arr);

and also async keyword should be removed from the function